### ISSUE TITLE
Support for iConsole+ connection - ZIPRO Notus

### ISSUE THREAD

**Issue by Deteorek** [cagnulein/qdomyos-zwift#1351]
Is it possible to run a connection between QZ and iConsole app? For now, I can't connect to this app. When I select Bluetooth device (my phone with QZ app) I've got an error message "Can't find the expected UUIDs" - Is there anything we can do with it?

Thanks

**Comment by cagnulein** [comment]
sorry @Deteorek are we ok with this? I don't remember

**Comment by Deteorek** [comment]
No, we not even started. I know you are busy so I waiting patiently 😀

**Comment by cagnulein** [comment]
ah i read again the main topic, yeah now i remember :)

**Comment by Deteorek** [comment]
Sorry, I bother you but any news? Can we do this or it's too complicated? I understand if not. Thanks.

**Comment by cagnulein** [comment]
@Deteorek it's not too complicated but it requires time (at least 1-2 days of work) and actually you're the only one interested on this, so I scheduled on low priority. As you can see I'm receiving a lot of requests each day so I have to prioritize things :)
